docs: reorganize README.md (#1054)
deepmodeling/abacus-develop: README.md

Please refer to our [GitHub repository](https://github.com/deepmodeling/abacus-develop) for more information and support.
# Table of contents

- [Features](#features)
- [Download and install](#download-and-install)
- [Quickstart guide](#quickstart-guide)
  - [Input files](#input-files)
  - [Run ABACUS](#run-abacus)
  - [Output files](#output-files)
  - [Hands-on examples](#hands-on-examples)
- [Citations](#citations)
- [Development team](#development-team)
- [Communicating and making contributions](#communicating-and-making-contributions)

# Features

ABACUS provides the following features and functionalities:

1. Three types of supported basis sets: pw, LCAO, and LCAO-in-pw.

> 2. Ground-state total energy calculations using Kohn-Sham (KS) density functional theory (DFT) with local-density, generalized gradient approximations (LDA/GGAs), Meta-GGA(depend on LIBXC, only for PW), and hybrid functionals (PBE0 and HSE06, only for LCAO and currently under test).

Reviewer: ```suggestion
2. Ground-state total energy calculations using Kohn-Sham (KS) density functional theory (DFT) with local-density, generalized gradient approximations (LDA/GGAs), Meta-GGA(requires LIBXC, only for PW), and hybrid functionals (PBE0 and HSE06, only for LCAO and currently under test).
```

> 3. Geometry relaxations with Conjugated Gradient (CG) and BFGS methods and FIRE methods.

Reviewer: ```suggestion
3. Geometry relaxations with Conjugated Gradient (CG), BFGS, and FIRE methods.
```

4. Semi-empirical van der Waals energy correction using the Grimme DFT-D2/D3 scheme.
5. NVT and NVE molecular dynamics simulation. AIMD, DP potential, LJ potential are supported.
6. Stress calculations and cell relaxations.
7. Electric polarization calculation using Berry Phase theory.
8. Interface to the Wannier90 package.
9. Real-time time dependent density functional theory (TDDFT).
10. Print-out of the electrostatic potential.
11. Mulliken charge analysis (only for LCAO).
12. Projected density of states (PDOS) (only for LCAO).
13. DFT+U calculation (only for LCAO).
14. Solvation model method for solvation energy.
15. Stochastic DFT (only for PW).

> 16. DeePKS method (trial version, only for LCAO).

Reviewer: "trial version" sounds like you should pay for it after 7 days 😂
```suggestion
16. DeePKS method (under development, only for LCAO).
```

17. Electric field and dipole correction.
18. Orbital-free DFT.
19. (subsidiary tool)Plot_tools for plot PDOS and PBANDS.

> 20. (subsidiary tool)Generator for second second generation numerical orbital basis.

Reviewer 2: double "second"

Reviewer: ```suggestion
20. (subsidiary tool)Generator for second generation numerical orbital basis.
```

    Specifications of the STRU file can be found in this [short instruction](docs/input-stru.md).
- `KPT`

    This is the k-point file that contains the information of the k-grid setting for the Brillouin zone sampling.
    Specification of the k-point file can be found in this [short instruction](docs/input-kpt.md).
- The pseudopotential files

    Norm-conserving pseudopotentials are used in ABACUS, in the UPF file format.The filename of each element’s pseudopotential needs to be specified in the STRU file, together with the directory of the pseudopotential files unless they are already present in the working directory.

    More information on pseudopotentials is given [here](docs/features.md#pseudopotentials).

- The numerical orbital files

    This part is only required in LCAO calculations. 
    The filename for each element’s numerical orbital basis needs to be specified in the STRU file, together with the directory of the orbital files unless they are already present in the working directory.

>     ABACUS provides numerical atomic basis sets of different accuracy levels for most elements commonly used. Users can download these basis sets from the [website](http://abacus.ustc.edu.cn/pseudo.html). Moreover, users can generate numerical atomic orbitals by themselves, and the procedure is provided in this [short introduction](docs/generate-basis.md).

Reviewer: @dyzheng dead link here
http://abacus.ustc.edu.cn/pseudo.html

Author: https://abacus.ustc.edu.cn/pseudo/list.htm
#should be this link but idk how to make commit on top of this conversation lol

Reviewer: ```suggestion
    ABACUS provides numerical atomic basis sets of different accuracy levels for most elements commonly used. Users can download these basis sets from the [website](http://abacus.ustc.edu.cn/pseudo/list.htm). Moreover, users can generate numerical atomic orbitals by themselves, and the procedure is provided in this [short introduction](docs/generate-basis.md).
```

    
- **If DeePKS is used:**

    [1]. Wenfei Li, Qi Ou, et al. "DeePKS+ABACUS as a Bridge between Expensive Quantum Mechanical Models and Machine Learning Potentials." 
    
- **If hybrid functional is used:**
    
    [1]. Peize Lin, Xinguo Ren, and Lixin He. "Efficient Hybrid Density Functional Calculations for Large Periodic Systems Using Numerical Atomic Orbitals." Journal of Chemical Theory and Computation 2021, 17(1), 222–239.
    
    [2]. Peize Lin, Xinguo Ren, and Lixin He. "Accuracy of Localized Resolution of the Identity in Periodic Hybrid Functional Calculations with Numerical Atomic Orbitals." Journal of Physical Chemistry Letters 2020, 11, 3082-3088.
    
[back to top](#readme-top)

# Development team
The current development team consists the following research groups/affiliations:
- University of Science and Technology of China (Dr. Lixin He)
- Peking University (Dr. Mohan Chen)
- Institute of Physics, Chinese Academy of Sciences (Dr. Xinguo Ren)
- AI for Science Institute

[back to top](#readme-top)

# Communicating and making contributions

> We also provide some [information](docs/CONTRIBUTING.md) on how to make contributions to ABACUS.

Reviewer: ```suggestion
If you find a bug or have some questions, please refer to our GitHub [issue tracker](https://github.com/deepmodeling/abacus-develop/issues), and our developers are willing to help. We also provide guidelines on how to make contributions to ABACUS.
```

[back to top](#readme-top)

# Citations
The following references are required to be cited when using ABACUS. Specifically:
- **For general purpose:**

    [1]. Mohan Chen, G. C. Guo, and Lixin He. "Systematically improvable optimized atomic basis sets for ab initio calculations." Journal of Physics: Condensed Matter 22.44 (2010): 445501.
    
    [2]. Pengfei Li, et al. "Large-scale ab initio simulations based on systematically improvable atomic basis." Computational Materials Science 112 (2016): 503-517.

- **If Stochastic DFT is used:**

    [1]. Qianrui Liu, and Mohan Chen. "Plane-Wave-Based Stochastic-Deterministic Density Functional Theory for Extended Systems." https://arxiv.org/abs/2204.05662
    
- **If DFT+U is used:**

    [1]. Xin Qu, et al. "DFT+ U within the framework of linear combination of numerical atomic orbitals." The Journal of Chemical Physics (2022).

- **If second generation numerical orbital basis is used:**

>     [1]. Peize Li, Xinguo Ren, and Lixin He. "Strategy for constructing compact numerical atomic orbital basis sets by incorporating the gradients of reference wavefunctions." Physical Review B 103.23 (2021): 235131.

Reviewer 3: Should be "Peize Lin"

Author: ```suggestion
    [1]. Peize Lin, Xinguo Ren, and Lixin He. "Strategy for constructing compact numerical atomic orbital basis sets by incorporating the gradients of reference wavefunctions." Physical Review B 103.23 (2021): 235131.
```
